Tests/MaterialLib/TestMPLSaturationVanGenuchten.cpp

```diff
@@ -21,11 +21,11 @@ TEST(MaterialPropertyLib, SaturationVanGenuchten)
     double const residual_liquid_saturation = 0.1;
     double const residual_gas_saturation = 0.05;
     double const exponent = 0.79;
-    double const entry_pressure = 5000;
+    double const alpha = 1./5000;
```

Comment:
Can use p_b directly.

Reply:
OK. I'll check the implementation and use p_b directly everywhere.